hooks/mari/collector.py

```diff
@@ -437,4 +455,7 @@ def _resolve_item_fields(self, settings, item):
             fields["layer"] = layer_name
             fields["UDIM"] = "FORMAT: $UDIM"
 
+            if item.type == "mari.mipmap":
+                fields["extension"] = "tx"
```

Comment:
This should be a plugin setting. There are a number of different mipmap formats that we may need to generate depending on what the consuming renderer is downstream. In fact, we may need to generate more than one (i.e. Lighting in vray, FX in mantra). We should probably reconfigure this plugin to add a `mari.mipmap` item per requested file type, and store that as an Item property at creation time and then plug it in here.

Reply:
Will rework this, and introduce a separate template as well, to separate mipmaps by folder (considering Deke's comment about files having the same extension)

Reply:
I can make the extension a plugin setting.

If I make it a list though, I will be producing multiple files, and only one Published File entity is created per item right now (which can have only one path), the path being set through `item.properties.publish_path` (though `publish_files()` returns a list, this is unused by the calling function)
If we rewrite `publish()` to use the result of this, I'm not sure what all it may lead to (where `item.properties.publish_path` is relied on). For example, in `symlink_publishes()`, we symlink publish_path to publish_symlink_path. If there are multiple paths, how does one derive multiple symlink paths?
So I'm reluctant to touch this...let me know what you think.

Reply:
I could spawn copies of the plugin per extension. Does that seem alright?

Reply:
Oops, I meant separate items per extension, not copies of the plugin.

Reply:
Ended up talking to myself...implemented item per extension.

Reply:
Yeah...I think that's the right way to go.